test_credentials/BUILD

```diff
@@ -6,30 +6,34 @@ gcloud_secret(
     name = "bigquery.json",
```

Comment:
if everybody in the world can decrypt these secrets, why bother keeping them as encrypted files?

Reply:
I think there is a big difference in leaving plain unencrypted secrets on the indexed web from obfuscating them. We'll at least know when someone decodes them if they send a PR, as that's the only way to do it.